go.mod

```diff
@@ -1,8 +1,8 @@
-module github.com/GoogleCloudPlatform/protoc-gen-bq-schema
+module github.com/GoogleCloudPlatform/protoc-gen-bq-schema/v2
 
-go 1.16
+go 1.21
```

Comment:
This is bumped because the minimum supported version in Go protobuf is `v1.21`

https://github.com/protocolbuffers/protobuf-go/blob/v1.35.2/go.mod#L3